styles.css

```diff
@@ -0,0 +1,126 @@
+body {
+  font-family: 'Poppins', sans-serif;
+}
+
+.header-2 {
+  background-image: url(img/BackGround.png);
+  font-style: normal;
+  background-repeat: no-repeat;
+  background-size: 100%;
+  margin: 0;
+  justify-content: center;
+  text-align: center;
+  height: inherit;
+  width: inherit;
+  left: -34px;
+  top: 103.9993896484375px;
+  border-radius: 0;
+  padding-top: 104px;
+}
```

Comment:
- [ ] Please I highly recommend adding `header-2` and `icons-section`  classes into one section and displaying them with CSS flexbox as it is part of the project requirements. 😃